chain/chain/src/store.rs

```diff
@@ -1791,6 +1782,7 @@ impl<'a> ChainStoreUpdate<'a> {
                 // Broken GC prerequisites found
                 assert!(false);
             }
+            // Don't clean Genesis Block
             return Ok(());
```

Comment:
`store_update` never gets committed if we just return

Reply:
I wonder which tests can help us to catch this types of bugs related to Genesis?

Reply:
we need a storage consistency check.
collect all records from all columns and check for leaks and dangling references

Reply:
It seems it's an Epic-size test.
Definitely should be done.